airframe-http-recorder/src/test/scala/wvlet/airframe/http/recorder/HttpRecorderTest.scala

```diff
@@ -123,17 +123,16 @@ class HttpRecorderTest extends AirframeSpec {
 
   }
 
-  "programmable server" in {
-    val response = withResource(HttpRecorder.createProgrammableServer { recorder =>
-      val request = Request("/index.html")
+  "programmable server" taggedAs working in {
+    val response = withResource(HttpRecorder.createInMemoryProgrammableServer) { server =>
+      server.clearSession
```

Comment:
This is unnecessary here as the database is an in-memory one, but added for test coverage.